packages/lodestar/src/sync/sync.ts

```diff
@@ -101,35 +130,6 @@ export class BeaconSync implements IBeaconSync {
     return this.state === SyncState.Synced;
   }
 
-  get state(): SyncState {
```

Comment:
Why all this changes in a PR that's clearly scoped to "import order"? Could you please revert this commit and do this changes if applicable? We have really never followed the member-ordering rule in Lodestar